tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/WebRequest20Tests.cs

```diff
@@ -16,6 +16,8 @@ namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
     public class WebRequest20Tests : TracingIntegrationTest
     {
+        private const string ServiceName = "Samples.WebRequest.NetFramework20";
```

Comment:
nit: this is available without duplication as `this.EnvironmentHelper.FullSampleName`

Reply:
Thanks! Fixed in b3ee46bca9a6e87cdcfb5b2c1b67cc2d0e59e1ea